.github/workflows/maven-graalvm.yml

```diff
@@ -0,0 +1,37 @@
+name: GraalVM Build
```

Comment:
This should be `GraalVM Maven Build` so it is easier to distinguish in the GitHub Actions Web UI

Reply:
Yes good point will address that